Include/cpython/pystats.h

```diff
@@ -118,7 +118,7 @@ typedef struct _optimization_stats {
     uint64_t recursive_call;
     uint64_t low_confidence;
     uint64_t executors_invalidated;
-    UOpStats opcode[MAX_UOP_ID];
+    UOpStats opcode[MAX_UOP_ID+1];
```

Comment:
Yeah, this is why I didn't like that constant being the largest used value instead of the next free one. :-)